app/models/legal_aid_application.rb

```diff
@@ -527,9 +527,17 @@ def hmrc_response_use_case_one
   end
 
   def uploading_bank_statements?
+    client_not_given_consent_to_open_banking? || attachments.bank_statement_evidence.exists? || attachments.partner_bank_statement_evidence.exists?
+  end
+
+  def client_uploading_bank_statements?
     client_not_given_consent_to_open_banking? || attachments.bank_statement_evidence.exists?
   end
 
+  def partner_uploading_bank_statements?
+    attachments&.partner_bank_statement_evidence&.exists?
```

Comment:
I am not familiar with the rules here, however, why safe navigation for partner upload, but not the client?

Reply:
Because the client version has the open_banking step... 
If the client says no to open banking, they _have_ to upload bank statements
Partners can be a bit more open 🤷‍♂️

Reply:
The alternative would be...
Is there a partner? Are they employed? Have they uploaded their bank statments?
```@legal_aid_application.has_partner? && @legal_aid_application.partner.employed? && attachments.partner_bank_statement_evidence.exists?```